File: 897662-lcm-of-two-numbers-GFG/897662-lcm-of-two-numbers.cpp
class Solution {
  public:
    int lcm(int a, int b) {
        
        for(int i=1;i>0;i++){
            if(i%a==0 && i%b==0){
                return i;
            }
        }
        return 0;
    }
};

// Synced seamlessly with LeetHub Pro
// Pro features: https://bit.ly/leethubpro | Free version: https://bit.ly/leethubv4
// Get it here: https://chromewebstore.google.com/detail/bcilpkkbokcopmabingnndookdogmbna

File: 897662-lcm-of-two-numbers-GFG/897662-lcm-of-two-numbers.py
class Solution:
       
    def lcm(self, a, b):
        c = abs(a*b)
        
        while b:
            a,b = b, a%b
            
        return c//a

# Synced seamlessly with LeetHub Pro
# Pro features: https://bit.ly/leethubpro | Free version: https://bit.ly/leethubv4
# Get it here: https://chromewebstore.google.com/detail/bcilpkkbokcopmabingnndookdogmbna